internal/configurations/formatting.handler.ps1

```diff
@@ -0,0 +1,121 @@
+﻿#region Formatting.Date
+$ScriptBlock = {
+    Param (
+        [string]
+        $Value
+    )
+    
+    $Result = New-Object PSOBject -Property @{
+        Success = $True
+        Message = ""
+    }
+    
```

Comment:
Well ... given that this is a hashtable declaration ... yes, there is a reason:
It's a technical requirement.
I could work around this by escaping the linebreak symbol (that's what the final backtick in a line does), but that's not neat and I really don't like people confusing Scriptblock/Hashtable declarations with braces denoting loops or scope-boundaries (such as when declaring a function). There's a vast technical difference right there.

Either way, the property-parameter expects a hashtable, so I give it one and start specifing it right behind it, which is where PowerShell expects values bound to a parameter.